packages/run/src/commands/logs.ts

```diff
@@ -0,0 +1,42 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {ProcessTypeCompletion} from '@heroku-cli/command/lib/completions'
+
+import logDisplayer from '../lib/log-displayer'
+
+export default class Logs extends Command {
+  static description = `display recent log output
+disable colors with --no-color, HEROKU_LOGS_COLOR=0, or HEROKU_COLOR=0`
+
+  static examples = [
+    '$ heroku logs'
```

Comment:
It helps to have the examples contain different combinations of flags to get a quick sense of what's possible.

Reply:
Added several more examples